### DISCUSSION TITLE
Outline similar to RDi

### DISCUSSION THREAD

**Discussion by WmARapp** [codefori/vscode-ibmi#329] [Ideas]
content assist is fantastic but an outline would be an added feature to take it to another level

**Comment by worksofliam** [comment]
Outline view for RPGLE is available in the normal VS Code Explorer and you can move it to another view to make it more convenient.

![image](https://user-images.githubusercontent.com/3708366/134712557-1c909ca3-a6c7-4a2b-99d3-8b9df7a20b56.png)
